### PR TITLE
Improve A11y

### PR DESCRIPTION
**Pull request by pvagner**
The app uses standard android UI controls and it's almost fully accessible for screen reader users. The minor accessibility issue is that image only buttons are missing clear textual content descriptions. I'm trying to improve this. I have tested the app with Talkback on a real device.

### CODE REVIEW

File: app/build.gradle
Comment (morckx) [inline app/build.gradle:1]: This change seems unrelated to the PR, please remove it.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     You should have received a copy of the GNU General Public License
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en" tools:ignore="MissingTranslation">
```
Comment (morckx) [inline app/src/main/res/values/strings.xml:18]: Hmm, I'd rather not ignore missing translations, just for the sake of completing A11y more easily. I could also just add the missing translations.

What do you think, @udenr ?

**Review by morckx — CHANGES_REQUESTED** [review]
Thanks!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (pvagner) [reply app/src/main/res/values/strings.xml:18]: So you would rather copy english original strings into each translation specific xml file when new strings are added?
Otherwise lint is not happy and each check is failing.

File: app/build.gradle
Comment (pvagner) [reply app/build.gradle:1]: Oops, that was my oversight. Hopefully fixed.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (udenr) [reply app/src/main/res/values/strings.xml:18]: You could set the MissingTranslation to be treated as warning instead of error. Example [here](https://github.com/SecUSo/privacy-friendly-todo-list/blob/a228ecda2627b85eab79929156274b406bfb1807/app/build.gradle#L56-L58).
